### PR TITLE
Add env vars for user & passive ports

### PR DESCRIPTION
**Pull request by GerkinDev**
Hi,

I've added some capabilities to your container, to use env-defined user & passive ports. Readme was completed with new options.

Related to #66

Cheers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,26 @@ Operating it
 
 `docker exec -it ftpd_server /bin/bash`
 
+User
```
[the remark this answers is not in the document]
Comment (GerkinDev) [reply README.md]: Yep, ofc. I've added another small feature in the meantime to set up the user uid & gid. Will commit this asap too ;)

File: run.sh [hunk 2]
```diff
@@ -24,6 +26,30 @@ then
...
+then
+    echo "Creating user..."
+    PWD_FILE="$(mktemp)"
+    echo "$FTP_USER_PASS
+$FTP_USER_PASS" > "$PWD_FILE"
+    pure-pw useradd "$FTP_USER_NAME" -f "$PASSWD_FILE" -m -u ftpuser -d "$FTP_USER_HOME" < "$PWD_FILE"
+    rm "$PWD_FILE"
+else
+    echo "Please set FTP_USER_NAME, FTP_USER_PASS & FTP_USER_HOME env"
+    exit 1
```
[the remark this answers is not in the document]
Comment (GerkinDev) [reply run.sh]: Indeed. I did this requirement for my personal use, but your container should be as generic as possible. Thus, i'll remove this `else` clause

File: run.sh [hunk 3]
```diff
@@ -7,14 +7,16 @@ PURE_FTPD_FLAGS="$@ $ADDED_FLAGS "
 # start rsyslog
 if [[ "$PURE_FTPD_FLAGS" == *" -d "* ]] || [[ "$PURE_FTPD_FLAGS" == *"--verboselog"* ]]
 then
-	echo "Log enabled, see /var/log/messages"
-	rsyslogd
+    echo "Log enabled, see /var/log/messages"
+    rsyslogd
 fi
 
+PASSWD_FILE="/etc/pure-ftpd/passwd/pureftpd.passwd"
```
[the remark this answers is not in the document]
Comment (GerkinDev) [reply run.sh]: I don't see the interest of this: there's nothing that may take advantage of overriding this setting , as far as I can tell. If you think it should be overridable, then I'll add a test condition to set up the default value

[the remark this answers is not in the document]
Comment (stilliard) [reply run.sh]: Sure that makes sense, only mentioned it as I noticed you abstracted this to a variable :+1:

**Comment by GerkinDev** [comment]
Would you like me to push another doc page containing a demo `docker-compose` file to manage something, like a `wordpress` site, via your container? It would be from an actually working stack I already use in pre-prod

**Comment by stilliard** [comment]
Sure that sounds like a fantastic addition to help people get started faster thanks

**Comment by GerkinDev** [comment]
Here we are!

Changes are done, and I've created a [wiki page](https://github.com/stilliard/docker-pure-ftpd/wiki/Docker-stack-with-Wordpress-&-FTP) for the demo.

Tell me if you need anything else ;)

Cheers!

**Comment by stilliard** [comment]
Awesome, thank you for these changes :)

**Pull request merged**